meson.build

```diff
@@ -39,26 +39,36 @@ if not headers_only
 	libc_deps += frigg_dep
 	rtdl_deps += frigg_dep
 
-
 	add_languages('c', 'cpp')
 	c_compiler = meson.get_compiler('c')
 
 	add_project_arguments('-nostdinc', '-fno-builtin', language: ['c', 'cpp'])
 	add_project_arguments('-std=c++17', language: 'cpp')
 	add_project_arguments('-fno-rtti', '-fno-exceptions', language: 'cpp')
+	add_project_arguments('-isysroot=/home/computerfido/.local/share/lemon/sysroot/system', language: ['c', 'cpp'])
```

Comment:
this is a problem

Reply:
Ahh that I removed after I noticed I left that in